lte/gateway/c/session_manager/UEShard.h

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <string>
+#include <utility>
+#include <vector>
+#include <map>
+
+namespace magma {
+
+class UEShard {
```

Comment:
I think this change needs a C++ face lift also I will not start the class with private.

Reply:
What is a C++ face lift?

Reply:
Just more C++ best practices friendly